app/models/conversion_host.rb

```diff
@@ -211,7 +211,7 @@ def enable_conversion_host_role(vmware_vddk_package_url = nil, vmware_ssh_privat
       :v2v_transport_method => source_transport_method,
       :v2v_vddk_package_url => vmware_vddk_package_url,
       :v2v_ssh_private_key  => vmware_ssh_private_key,
-      :v2v_ca_bundle        => resource.ext_management_system.connection_configurations['default'].certificate_authority
+      :v2v_ca_bundle        => openstack_tls_ca_certs || resource.ext_management_system.connection_configurations['default'].certificate_authority
```

Comment:
Hey @fdupont-redhat @djberg96 couple of things:

First:
`resource.ext_management_system.connection_configurations['default'].certificate_authority`

Unless I'm reading this wrong, that looks like it can be replaced by `resource.ext_management_system.certificate_authority` because we delegate `:certificate_authority` to the `:default_endpoint` which is all the `connection_configurations['default']` does but you should try this yourself, untested

Second:
Why don't we just allow users to enter their certificate authority to the OSP provider and continue to do this in a provider agnostic way?

Reply:
> Hey @fdupont-redhat @djberg96 couple of things:
> 
> First:
> `resource.ext_management_system.connection_configurations['default'].certificate_authority`
> 
> Unless I'm reading this wrong, that looks like it can be replaced by `resource.ext_management_system.certificate_authority` because we delegate `:certificate_authority` to the `:default_endpoint` which is all the `connection_configurations['default']` does but you should try this yourself, untested

Nice one. I'll test in a follow-up PR.

> Second:
> Why don't we just allow users to enter their certificate authority to the OSP provider and continue to do this in a provider agnostic way?

The UI doesn't allow you to enter the certificate authority. We'll create an RFE. Unless you know how to do it and we missed it :)

Reply:
I doubt you missed it, but given that RHV does this already it sounds like an easily solvable problem

Reply:
Yes, RHV does it but I'm not sure how the backend would handle this. Today, to enable SSL validation with custom CA, you need to install the CA certs on the appliance manually (`update-ca-trust`), which is not very convenient at scale. We started the conversation on ManageIQ/manageiq-providers-openstack channel on Gitter.

Reply:
Usually the API clients (aka fog) allow you to pass your CA when you initiate the connection so that you don't have to modify the system certificate authorities.

Since we don't have an issue with openshift refresh I'm going to assume they just allow self signed certs without verification (@aufi correct me if I'm wrong here) so this would be specifically for v2v.

Reply:
I think that the main issue is that users will expect that the CA certificate is used by OpenStack provider code and that they don't have to install it manually. So, it's more about setting the wrong expectations with the new field.

Reply:
And we can enable that in the future, I want to avoid having multiple ways of doing things.

Reply:
Me too. That's definitely an RFE, but it requires more work to identify how it is leveraged by fog-openstack. Comment from @aufi on Gitter:

> I wasn't able display ExtManagementSystem.connection_configuration['default'] which doesn't work for me on master (and ExtManagementSystem.connection_config contains MIQ database settings, if ovirt uses it also for provider connection settings, it might be OK, I just didn't know it).
> Looking on fog-openstack (and excon which is used as HTTP client), there is :ssl_ca_path expecting path with CA files what means that CA certs should be stored in filesystem. I might have missed something, but so far I haven't found an easy way to pass CA cert content from MIQ DB to fog-openstack API calls, so it looks to be more complex to me.

Reply:
@agrare I've created the RFE to keep track of this.

Reply:
Okay I'm alright with this as is but really really really (3 reallys means it is serious) don't want to forget about moving this out of here and to the provider